components/editor/geany/manifests/sample-manifest.p5m

```diff
@@ -243,6 +243,7 @@ file path=usr/share/locale/be/LC_MESSAGES/geany.mo
 file path=usr/share/locale/bg/LC_MESSAGES/geany.mo
 file path=usr/share/locale/ca/LC_MESSAGES/geany.mo
 file path=usr/share/locale/cs/LC_MESSAGES/geany.mo
+file path=usr/share/locale/da/LC_MESSAGES/geany.mo
```

Comment:
If the sample-manifest changes you'll need to update the .p5m file.
Otherwise new files will be missing and obsoleted files will still be delivered (if old version still lurks around otherwise it will create an error and will be detected).
This is the reason to always run 'gmake sample-manifest' after a version change.